rpa-builder/6.6/modules/ROOT/pages/toolbox-file-operations-read-pdf-file.adoc

```diff
@@ -1,58 +1,39 @@
-
-
 = Read PDF File
 
 With the _Read PDF File_ Action Step, you can read text from a PDF file.
 
 == Properties
 
-*File Settings*
-
-*Directory Path* (String)
-The path to the folder in which the PDF file is located.
-
-*File Name* (String) The name of the PDF file.
-
-*Password protected* (Boolean) This indicates whether the PDF document is to be opened with a
-password.
-
-*Read Mode*
-
-*Read Mode* Read mode for the PDF document.
-
-The following options all belong to the Read Mode _Read page range_:
+* *File Settings*
++
+** *Directory Path* (String): Path to the folder in which the PDF file is located.
+** *File Name* (String): Name of the PDF file.
+** *Password protected* (Boolean): Select this option if the PDF file is password protected. 
 
-* *Read text from page*
-(Integer) The text is read from this page onward.
-* *Read text to page*
-(Integer) The text is read up to this page.
-* *Read to end of file* (Boolean) If this option is selected, the text is
-read up to the end of the document.
+* *Read Mode*
+** *Read form field data and annotations*:  Select this option to read data from form fields and annotations in the PDF file. 
+** *Read Mode*: Read mode for the PDF document.

```

Comment:
Are there options? If so, would it make sense to list them here, or at least say .. such as ... and list a couple?